docs/server-centos6.md

```diff
@@ -1,42 +1,48 @@
-# Algo pre-install steps for Red Hat/CentOS 6.x (currently 6.8)
+# RedHat/CentOS 6.x server setup
```

Comment:
I would still call this "RedHat/CentOS 6.x pre-install steps." This is only intended to run the Algo ansible scripts, not host a server.